code-of-conduct.md

```diff
@@ -1,38 +1,134 @@
+- Deliberate misgendering, using inappropriate pronouns, or use of “dead” or
+rejected names
+- Harassing or non-consensual photography or recording
+- Harassment of event participants in any form
+- Hate symbols
+- Incitement of violence towards any individual or group, including encouraging
+a person to engage in self-harm
+- Insults, put downs or excessive swearing
+- Intimidation, stalking or following
+- Inappropriate physical contact or unwelcome contact of any kind, especially
+after a request to stop
+- Physical assault and/or battery
+- Policing bathrooms based on gender expression or identity
+- Publication of non-harassing private communication
+- Sexist, racist, homophobic, transphobic, ableist, or other exclusionary jokes
+- Sexual language and imagery in any event venue, including talks, Slack,
+Twitter, or other online media
+- Sustained disruption of talks or other events
+- Threats - violent or otherwise - directed against another person
+- Unwelcome comments regarding a person’s lifestyle choices and practices,
+including those related to food, health, parenting, relationships, drugs, and
+employment
+- Unwelcome sexual attention or advances
+- Other conduct that is inappropriate for a professional audience that includes
+people of many different backgrounds
+
+Participants asked to stop any inappropriate behavior are expected to comply
+immediately.
+
+If a participant engages in behavior that violates this code of conduct, the
+event organizers may take any action they deem appropriate, including warning
+the offender or expulsion from the event (without refund, if applicable) and
+banning from future events.
+
+Thank you for helping make this a welcoming, friendly community for all.
+
+### Weapons Policy
+
+No weapons are allowed at ChiPy venues, including but not limited to explosives
+(including fireworks), guns, and large knives such as those used for hunting or
+display, as well as any other item used for the purpose of causing injury or
+harm to others. Anyone seen in possession of one of these items will be asked
+to leave immediately, and will only be allowed to return without the weapon.
+
+Attendees are further expected to comply with all state and local laws on this
+matter.
+
+## Scope
+
+All ChiPy attendees are subject to the Code of Conduct. This includes ChiPy
+organizers and volunteers, as well as attendees of the talk events,
+collaborative work environments, receptions / after-party events, or other
+sponsored or unofficial events related to ChiPy events.
+
+Sponsors are also subject to the anti-harassment policy. In particular,
+sponsors should not use sexualised images, activities, or other material.
+
+## Contact Information
+
+Anyone can report harassment. If someone’s behavior has made you uncomfortable,
+or if you witness the same happening to someone else, you should immediately
+contact ChiPy organizers in-person or on Slack, or send an email to the On-Call
+responser at chipy.conduct@gmail.com.
```

Comment:
It probably is a minutiae, but instead of gmail, should we get a python.org email address like we have for organizers and the main mailing list? I can't find the email, but have a vague memory that we reached out to someone in who manages python.org mailing lists/listservs

Reply:
I think we should have a subcommittee instead of an on-call person

Reply:
We could get another email address; this is just the one we have right now. As soon as we get one we can replace.

As for the subcommittee, the proposal is that the Subcommittee gets emails forwarded from this account. It's not one person; it's just one email address.